xml/admin_ceph_gateway.xml

```diff
@@ -3234,7 +3239,35 @@ rgw_frontends="beast port=80"
 rgw_zone=us-east-2</screen>
 <screen>&prompt.cephuser;sudo systemctl start ceph-radosgw@rgw.us-east-2</screen>
    </sect3>
-  </sect2>
+ <sect3 xml:id="ceph-update-dashboard-config">
+   <title>Update Dashboard Configuration</title>
+   <procedure>
+     <step>
+       <para>Set the following to update your dashboard configuration for
+       multi-site:</para>
+       <screen>&prompt.cephuser; ceph dashboard set-rgw-api-access-key ACCESS-KEY
+       &prompt.cephuser; ceph dashboard set-rgw-api-secret-key SECRET-KEY
+      </screen>
```

Comment:
Your indentation in this case causes the second prompt to be indented. screen markup should be sort of pushed to the left. Run `daps xmlformat` and check its result before comitting. It's useful for unifying the diffs.

```
<screen>&prompt.cephuser; ceph dashboard set-rgw-api-access-key <replaceable>ACCESS-KEY</replaceable>
&prompt.cephuser; ceph dashboard set-rgw-api-secret-key <replaceable>SECRET-KEY</replaceable>
</screen>
```

Reply:
Good tip! Did'nt know that one :)